tests/runtime/performance/test_cli_perf.py

```diff
@@ -11,8 +11,10 @@
 https://zapatacomputing.atlassian.net/browse/ORQSDK-507
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Are these relevant now? Should they be switched to the new CLI?

Reply:
They do test that corq still works. I think we should keep them as long as we keep corq code.

Reply:
> Should they be switched to the new CLI?

Having similar ones for dorq sounds like a good idea, but I'd keep it out of scope. Added under https://zapatacomputing.atlassian.net/browse/ORQSDK-749